### PR TITLE
Changes for compatibility with Python 3

### PR DESCRIPTION
**Pull request by joachimmetz**
Changes for compatibility with Python 3

### CODE REVIEW

**Comment by PhillipSz** [comment]
Hey, can some please review this patch? Thank you!

File: gflags/argument_parser.py [hunk 1]
```diff
@@ -325,7 +325,7 @@ def __init__(self, list_sep):
 
   def Serialize(self, value):
     """Serialize a list as a string, if possible, or as a unicode string."""
-    output = cStringIO.StringIO()
+    output = io.StringIO()
     writer = csv.writer(output)
 
     # csv.writer doesn't accept unicode, so we convert to UTF-8.
```
Comment (barabo) [inline gflags/argument_parser.py]: I believe there is a lurking bug on the following line when running in python3 (use of `unicode` keyword).

Instead, I believe you can replace all occurrences (there's only one) of `CsvListSerializer()` with `ListSerializer(',')`.  This prevents the need for importing the `io` module entirely.

**Comment by gpshead** [comment]
take a look at the recent pushes, it should be working on Python 3.

**Comment by joachimmetz** [comment]
> take a look at the recent pushes, it should be working on Python 3.

Thanks I'll close this PR then.

**Pull request closed without merging**